### PR TITLE
fix #78869 asiatv.online

### LINKED ISSUE #78869: asiatv.online
### Issue URL (Anti Adblock Script)
 [https://asiatv.online](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fasiatv.online)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v1.10.3
Browser: | Edge
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion<br/><br/><b>Language-specific:</b><br/> Liste AR
Other extensions used: | Adblock for Youtube, global speed

### PR DESCRIPTION
**Pull request by wout-codes**
#78869

### CODE REVIEW

File: EnglishFilter/sections/foreign.txt [hunk 1]
```diff
@@ -33,6 +33,9 @@ video.gjirafa.com#%#AG_defineProperty('originVideo.requestA.Id', { value: "0" })
 !------------------------------ Arabian ------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/78869
+asiatv.*#$#.ad-banner { display: block !important; }
+asiatv.*#@#.ad-banner
```
Comment (zloyden) [inline EnglishFilter/sections/foreign.txt:38]: Why do you use `*` here?
`asiatv.online#@#.ad-banner` is enough.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/foreign.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/foreign.txt:38]: The domain changes quite frequently.

Comment (wout-codes) [reply EnglishFilter/sections/foreign.txt:38]: Changed to scriptlet. I don't like to unblock elements, unless necessary, for platforms which supports Scriptlets and JS.

Comment (zloyden) [reply EnglishFilter/sections/foreign.txt]: > Changed to scriptlet. I don't like to unblock elements, unless necessary, for platforms which supports Scriptlets and JS.

Save the scriptlet for the future, unblocking element is enough here.

**Review by zloyden — APPROVED** [review]

File: EnglishFilter/sections/foreign.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/foreign.txt]: Why did you remove it? I thought that's the way issues are supposed to be fixed. Check other anti-adblock fixes, it has a scriptlet + a whitelist rule for platforms without scriptlets support.

Comment (zloyden) [reply EnglishFilter/sections/foreign.txt]: I've already explained it. [Unblocking element is enough here](https://github.com/AdguardTeam/AdguardFilters/pull/78899#discussion_r604619883). No need to complicate things, use an easier way to solve issues.
Using the scriptlet will increase the load on the extension, you need to use it when there are no other options.

**Pull request merged**